### PR TITLE
Day3

### PR DESCRIPTION
**Pull request by Jyothi-Sittula**

### CODE REVIEW

File: BetweenTwoSets.cs [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        for(int i=left;i<=right;i++){
+            countA = a.Count(eachA => (i%eachA != 0));
+            if(countA != 0) continue;
+            countB = b.Count(eachB => (eachB%i != 0));
+            if(countB == 0)
+                total++;
+        }
+        return total;
+    }
+ */
```
Comment (narayanagunturi) [inline BetweenTwoSets.cs]: remove commented code

File: GradingStudents.cs [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+using System.Text.RegularExpressions;
+using System.Text;
+using System;
+
+class Result
+{
+
+    public static List<int> gradingStudents(List<int> grades)
+    {
+        //return grades.Select(grade => (!(grade<38)&&grade%5 >2)?grade+(5 - grade%5):grade).ToList();
```
Comment (narayanagunturi) [inline GradingStudents.cs]: remove commented code

File: TimeConversion.cs [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+using System.Runtime.Serialization;
+using System.Text.RegularExpressions;
+using System.Text;
+using System;
+
+class Result
+{
+    public static string timeConversion(string s)
+    {
+        //return Convert.ToDateTime(s).ToString("hh:mm:sstt"); reverse answer tt->AM,PM
```
Comment (narayanagunturi) [inline TimeConversion.cs]: remove commented code

**Review by narayanagunturi — CHANGES_REQUESTED** [review]

File: BetweenTwoSets.cs (hunk 1 above)
Comment (Jyothi-Sittula) [reply BetweenTwoSets.cs]: done @narayanagunturi

File: GradingStudents.cs (hunk 2 above)
Comment (Jyothi-Sittula) [reply GradingStudents.cs]: removes commented code @narayanagunturi

File: TimeConversion.cs (hunk 3 above)
Comment (Jyothi-Sittula) [reply TimeConversion.cs]: did it @narayanagunturi

**Pull request merged**